File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cs
public class Solution {
    public int RemoveDuplicates(int[] nums) {
        int j = 0;
        for(int i = 0 ; i < nums.Length-1; ++i){
            if(nums[i] == nums[i+1]){
                continue;
            }else{
                nums[j] = nums[i];
                ++j;
            }
        }
        nums[j] = nums[nums.Length-1];
        return j+1;
    }
}

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int i = 1, j = 1, n = nums.size();
        while(i<n && j<n){
            if(nums[j] == nums[j-1]) ++j;
            else{
                nums[i] = nums[j];
                ++i;
                ++j;
            }
        }
        return i;
    }
};